Python/ast.c

```diff
@@ -21,7 +21,7 @@ static int validate_stmt(stmt_ty);
 static int validate_expr(expr_ty, expr_context_ty);
 
 static int
-validate_name(PyObject *name)
+validate_name(const char* owner, PyObject *name)
 {
```

Comment:
In validate_name owner is the first arg and in validate_import_names it is the last arg.

Reply:
Other arguments besides `owner` are different in `validate_name()` and `validate_import_names()`, so I do not see a problem here.